services/src/main/java/org/exoplatform/officeonline/OfficeOnlineEditorService.java

```diff
@@ -681,10 +701,11 @@ protected void addFileURLProperties(Map<String, Serializable> map,
                                                              .append(PortalContainer.getCurrentRestContextName())
                                                              .toString();
 
-      // TODO: implement REST endpoint
       String downloadURL = new StringBuilder(platformRestURL).append("/officeonline/editor/content/")
-                                                             .append(WCMCoreUtils.getRemoteUser())
+                                                             .append(node.getUUID())
                                                              .append("/")
+                                                             .append(WCMCoreUtils.getRemoteUser())
```

Comment:
Use ConversationState.getCurrent(). Here we don't want tight on ECMS(WCM).

Reply:
Fixed in https://github.com/exo-addons/office-online/pull/2/commits/1a2933bb7099ed744368596f3d724b7396b29bd2